### PR TITLE
[packaging] Add 2023.03.02-3 release notes to .metainfo.xml

### PR DESCRIPTION
**Pull request by Ferenc-**

### CODE REVIEW

File: packaging/app.organicmaps.desktop.metainfo.xml [hunk 1]
```diff
@@ -113,6 +113,26 @@
   </screenshots>
 
   <releases>
+   <release version="2023.03.02-2" date="2023-03-02">
+     <description>
+       <p>Highlights:</p>
+         <ul>
+           <li>New OpenStreetMap data as of February 27</li>
+           <li>Wiki articles are now clickable and visible for bookmarks</li>
```
Comment (biodranik) [inline packaging/app.organicmaps.desktop.metainfo.xml:121]: It's android one

File: packaging/app.organicmaps.desktop.metainfo.xml [hunk 2]
```diff
@@ -113,6 +113,26 @@
...
+           <li>Fixed some not displayed complex opening hours</li>
```
Comment (biodranik) [inline packaging/app.organicmaps.desktop.metainfo.xml:122]: Android one

**Review by biodranik — COMMENTED** [review]
• New OpenStreetMap data as of February 27
• Fixed and improved search
• Split vegan and vegetarian cuisines in search
• Fixed styles for airports, land uses, waterways, and hatching fill areas
• Some smaller areas are not obscured by large ones anymore
• Added healthcare specialists: alternative, audiologist, blood_donation, optometrist, podiatrist, psychotherapist, sample_collection, speech_therapist
• Added communication towers, loading docks, solar, wind, gas, and hydropower generators
• Fixed the Bilbao metro icon
• Historic ruins can now be added in the Editor
• Improved Arabic, Norwegian, Portuguese, and Turkish translations

File: packaging/app.organicmaps.desktop.metainfo.xml (hunk 1 above)
Comment (Ferenc-) [reply packaging/app.organicmaps.desktop.metainfo.xml:121]: OK removed.

File: packaging/app.organicmaps.desktop.metainfo.xml (hunk 2 above)
Comment (Ferenc-) [reply packaging/app.organicmaps.desktop.metainfo.xml:122]: Removed

**Pull request merged**